### ISSUE TITLE
kiali crashes due to go 1.14 - move to go 1.14.1

### ISSUE THREAD

**Issue by jmazzitelli** [kiali/kiali#2779]
See: https://github.com/istio/istio/issues/21672

This needs to change: https://github.com/kiali/kiali/blob/v1.15.2/Makefile#L32

```
GO_VERSION_KIALI = 1.14
```

Needs to be 1.14.1 minimum.

**Comment by jmazzitelli** [comment]
Stable version today is 1.14.2 - will move to that.

**Comment by jshaughn** [comment]
Also reported on Istio Kiali Slack channel:

From Istio Kiali Slack:

We've been experiencing stability issues with kiali on our development machines since version 1.5.0 of istio. We've also tried 1.5.1, and today 1.5.4.

```
runtime: mlock of signal stack failed: 12
runtime: increase the mlock limit (ulimit -l) or
runtime: update your kernel to 5.3.15+, 5.4.2+, or 5.5+
fatal error: mlock failed
```
followed by a panic, and errors from various goroutines.
Is this the right place to highlight the issue, or is it better to go through github?
We're not doing anything particular when this happens. It crashes on its own without us accessing it after a short time running. If we access the dashboard through istioctl dashboards kiali it crashes instantly.

**Comment by israel-hdez** [comment]
This issue was tracked in Go side in https://github.com/golang/go/issues/37436.

In that thread it was said:
* it's a Kernel bug: https://github.com/golang/go/issues/37436#issuecomment-591327351
* Go1.14.0 already tries to mitigate the bug, but mitigation sometimes fails: https://github.com/golang/go/issues/37436#issuecomment-591237929
* More mitigation was added in 1.14.1: https://github.com/golang/go/issues/37436#issuecomment-597360484
* It's not an issue tied to Go, any app can fall in the bug: https://github.com/golang/go/issues/37436#issuecomment-591461934
* the Kernel bug can lead to memory/data corruption: https://github.com/golang/go/issues/37436#issuecomment-597990714 and https://github.com/golang/go/issues/37436#issuecomment-601766098

I think we should first point to the user that his Kernel is buggy. Not sure if it's worth back-porting the bump of the go version, given that Go is just doing a workaround of a Kernel bug.

**Comment by Multiply** [comment]
I'll be happy to test this on my system, as soon as a release is available.

**Comment by israel-hdez** [comment]
@Multiply Kiali v1.18.1 is now built agains go 1.14.2.
We still haven't released any new 1.15, so you would need to upgrade.
